src/backend/loader.js

```diff
@@ -1,17 +1,16 @@
-// @ts-check
-
 // This file sets up some global variables that are incorrectly set in
 // nodejs-mobile, e.g. the cwd points to root ("/") on mobile, so we override it
 // with the nodejs project dir
 
-const os = require('os');
-const path = require('path');
-const rnBridge = require('rn-bridge');
+import os from 'os'
+import path from 'path'
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+const rnBridge = require('rn-bridge')
```

Comment:
was getting complaints that `rn-bridge` isn't an ESM module (not transformed when bundling since we mark it as external for rollup), which is true:

https://github.com/nodejs-mobile/nodejs-mobile-react-native/tree/v16.17.10/install/resources/nodejs-modules/builtin_modules/rn-bridge